slack-api-client/src/main/java/com/slack/api/methods/impl/MethodsClientImpl.java

```diff
@@ -2934,7 +2934,10 @@ public AdminAnalyticsGetFileResponse adminAnalyticsGetFile(AdminAnalyticsGetFile
             if (response.isSuccessful()) {
                 try {
                     Gson gson = GsonFactory.createSnakeCase(slackHttpClient.getConfig());
-                    return gson.fromJson(body, AdminAnalyticsGetFileResponse.class);
+                    AdminAnalyticsGetFileResponse apiResponse =
+                            gson.fromJson(body, AdminAnalyticsGetFileResponse.class);
+                    apiResponse.setHttpResponseHeaders(toLowerCasedKeyMap(response.headers()));
```

Comment:
Only this method is different from others because the endpoint can return either text body or binary data. Thus, I repeated the same logic.